tests/src/integration/zipaction/zipaction_test.go

```diff
@@ -0,0 +1,28 @@
+// +build integration
+
+package tests
+
+import (
+	"github.com/apache/incubator-openwhisk-wskdeploy/tests/src/integration/common"
+	"github.com/stretchr/testify/assert"
+	"os"
+	"testing"
+)
+
+
+var wskprops = common.GetWskprops()
+
+// TODO: write the integration against openwhisk
```

Comment:
@pritidesai What is left to do here?  If there is more to be done I assume another issue or PR will be incoming.  i would like to get the base64 encoding merged in the meantime.